test/shell/move-upper-dir.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+# TEST FOR #124 --- Disabled for now
```

Comment:
Can you change the commit message prefix to "test: file:" please or "test: resources: file:" ... ?
The shell script could have a file- prefix as well.

Thanks